test/BUILD.bazel

```diff
@@ -106,13 +106,13 @@ rollup_bundle(
     entry_points = {
         ":test_bundling.ts": "index",
     },
+    format = "cjs",
```

Comment:
Make this `umd` so it can be loaded in both node and the browser.

You'll also need to add to the rollup config:

```
  output: {
    name: "test_es6_bundling",
  },
```